### DISCUSSION TITLE
Mail di promemoria per segnalazione chiusa?

### DISCUSSION THREAD

**Discussion by maelaborazioni** [globaleaks/globaleaks-whistleblowing-software#4010] [Q&A]
Buongiorno,
ho la seguente (unica) segnalazione tra i log:

| # | Data             | Ultimo agg.      | Scadenza   | Canale  | Stato  | Tor | Commenti | Allegati | Destinatari | Ultimo acc. segnalante |
|---|------------------|------------------|------------|---------|--------|-----|----------|----------|-------------|------------------------|
| 1 | 22-12-2023 09:41 | 22-12-2023 09:44 | 21-12-2028 | Default | Chiusa | X   | 1        | 0        | 2           | 22-12-2023 09:45       |

Il cliente mi segnala la ricezione di email di promemoria tutti i giorni a partire dal giorno **22-02-2024** alle ore **00:05** (e i log del mail server lo confermano)
Nelle impostazioni del **Canale c'è Data del promemoria (giorni)** = **60**

La mia domanda è: _queste email di promemoria non dovrebbero smettere di essere inviate una volta che la segnalazione è chiusa?_
Grazie

**Comment by maelaborazioni** [comment]
Ho notato una cosa che forse potrebbe aiutare a capire il motivo di questo invio di mail.
Ho creato infatti un utente ricevente di prova e dalla dashboard ho visto che uno degli altri riceventi ha impostato un reminder.
Credo sia per questo che quel ricevente riceve le notifiche via email.
Quello che non mi aspettavo è che le riceva anche nel caso di segnalazione chiusa. È corretto che sia così?

**Comment by evilaliv3** [comment]
Buongiorno @maelaborazioni 

Grazie per questa serie di domande.

Al momento promemoria e chiusura delle segnalazioni sono implementate come due funzionalità a se stanti in quanto si è ipotizzato che uno voglia tenere attivo un promemoria di controllare una segnalazione anche a segnalazione chiusa.

A mio avviso si richierebbe a mio avviso che un utente ipotizzando le funzionalità coma indipendenti si imposti un promemoria per poi scoprire a posteriori che in caso di segnalazioni chiuse  questa funzionalità non funziona.

Una alternativa potrebbe essere quella di resettare il promemoria automaticamente quando la segnalazione viene chiusa permettendo dunque ad un utente di ri-abilitarsi un promemoria manualmente.

Sollevo la tematica ad altri utenti esperti per vedere cosa suggeriscono: @rglauco @gianlucagilardi @giorgiofraschini @larrykind

**Reply by giorgiofraschini** [reply]
Io penso che il sistema attuale sia funzionale. Un utente dovrebbe aver coscienza di cosa sia un promemoria quando lo imposta. la stessa dicitura compare anche nelle notifiche.

**Comment by larrykind** [comment]
> Una alternativa potrebbe essere quella di resettare il promemoria automaticamente quando la segnalazione viene chiusa permettendo dunque ad un utente di ri-abilitarsi un promemoria manualmente.

Concordo.

**Reply by larrykind** [reply]
Ad esempio, come fare se un ricevente che non può riaprire segnalazioni chiuse dimentica di togliere il promemoria? Attendere la scadenza della segnalazione non credo sia una soluzione.

**Comment by evilaliv3** [comment]
@larrykind : la modifica del promemoria è permessa anche su segnalazioni chiuse

inoltre ora che ci penso i promemoria sono personali quindi troverei rischioso che un utente che vari uno stato di una segnalazione finisca per disabilitare un promemoria di un altro utente perchè è un comportamento che l'altro utente potrebbe non aspettarsi.

Per questo motivo anche io come da parere di  @giorgiofraschini  rimango dell'idea che l'implementazione corrente sia la più funzionale discussa

**Comment by maelaborazioni** [comment]
Grazie per le vostre risposte.
Per darvi un ulteriore riscontro vi dico che ho già ricevuto 4 o 5 casi di clienti che mi segnalano che gli arriva la mail di promemoria nonostante la segnalazione sia stata chiusa. 
Ho spiegato loro la logica che ci sta dietro ovvero quella di cui avete discusso anche voi in questo thread.
Detto ciò mi metto nei loro panni e comprendo che possa risultare controintuitiva.
Forse per _salvare capra e cavoli_ si potrebbe pensare di parametrizzare questo comportamento/automatismo.
